### PR TITLE
feat(cryo): add G10-TUPLE-HE-REF runtime governance validation core

### PR DESCRIPTION
**Pull request by GBOGEB**
### Motivation
- Integrate a lightweight, SSOT-driven cryogenic validation engine to enforce thermodynamic invariants for the G10-TUPLE-HE-REF runtime target.
- Ensure mass-fraction parity and 2K-mode performance checks with deterministic four-decimal precision suitable for high-precision validation pipelines.

### Description
- Add `physics/helium_refrigeration_core.py` which implements `verify_mass_fractions`, `validate_2k_static_benchmark`, `validate_2k_operational_flow`, and `run_governance_assimilation` that reads SSOT targets and enforces runtime gates.
  - Uses `Decimal` arithmetic with `quantize()` for deterministic 4-decimal precision in mass fraction validation (handles edge cases like `[0.1]*10` correctly)
  - Comprehensive YAML schema validation with explicit error messages for empty files, missing keys, and malformed data
  - Returns `bool` for testability and safe imports (exit code conversion only in `__main__` path)
  - Path resolution via `pathlib` relative to `__file__` for reliable execution from any directory
- Add `SSOT/g10_runtime_governance_ssot.yaml` containing the minimal SSOT component `G10-TUPLE-HE-REF` with `2K-SB.target_efficiency` and `2K-OP.nominal_flow_g_s` entries.
- Add `METRICS/g10_ascii_metrics.md` providing ASCII telemetry summary and gate-status report with accurate validation gate descriptions matching actual repo checks (pytest + governance scripts).
- Add `tests/test_helium_refrigeration_core.py` with comprehensive test suite (29 test cases) covering pass/fail boundaries, edge cases, and malformed SSOT scenarios.
- Keep the runtime implementation dependency-free aside from `PyYAML` and include a runnable smoke path in `__main__` for local verification.

### Testing
- Ran `python physics/helium_refrigeration_core.py` which executed the smoke path and reported all gates PASS (script exited successfully with the governance gate PASS).
- All 29 pytest test cases pass, covering:
  - Mass fraction validation with Decimal precision (exact, rounding, overflow/underflow, determinism)
  - 2K-SB efficiency gates (pass/fail boundaries)
  - 2K-OP flow gates (tolerance boundaries)
  - SSOT parsing robustness (missing file, empty file, non-dict YAML, syntax errors, missing keys, wrong types)
  - Full governance assimilation with all gate combinations
- Verified the new files were staged and committed successfully during the rollout (files created: `physics/helium_refrigeration_core.py`, `SSOT/g10_runtime_governance_ssot.yaml`, `METRICS/g10_ascii_metrics.md`, `tests/test_helium_refrigeration_core.py`).
- All repository governance checks pass (manifest, globs, stale, links, render linter).

------
<a href="https://chatgpt.com/codex/cloud/tasks/task_b_6a14ec1ba1e8832eba522ba2edcf4bb1">Codex Task</a>

### CODE REVIEW

File: METRICS/g10_ascii_metrics.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+SSOT YAML Specification       [####################] 100%
+Workflow Verification Workers [####################] 100%
+Cryo Thermodynamic Core (HE)  [####################] 100%
+ASCII Log Engine              [####################] 100%
+Hosted HTML Avoidance Factor  [####################] 100%
+
+## Automated Gate Status
+[PASS] WCAG Contrast Gate      (Target >= 4.5:1 | Actual: 4.69:1)
+[PASS] Smoke Test Environment  (Orchestration script bypass verified)
+[PASS] Lint Verification Gate  (Black/Flake8 layout check)
```
[the remark this answers is not in the document]
Comment (GBOGEB) [reply METRICS/g10_ascii_metrics.md]: @copilot fix and commit

File: physics/helium_refrigeration_core.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import yaml
+
+
+def verify_mass_fractions(fractions: list) -> bool:
+    """Enforces absolute precision validation for fluid stream components to 4 decimal places."""
+    total = sum(round(f, 4) for f in fractions)
+    expected = round(1.0000, 4)
+    if total != expected:
+        print("CRITICAL FAULT: Stream mass fraction cumulative totals deviate from SSOT boundary condition.")
+        print(f"Expected: {expected:.4f}, Calculated: {total:.4f}, Delta: {abs(expected - total):.4f}")
```
[the remark this answers is not in the document]
Comment (GBOGEB) [reply physics/helium_refrigeration_core.py]: @copilot fix and commit

File: physics/helium_refrigeration_core.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    print(
+        f"[Mode: 2K-OP] Target Flow: {target_flow:.4f} g/s "
+        f"(Tol: {tolerance * 100:.1f}%) | Detected: {nominal_flow:.4f} g/s"
+    )
+    return lower_bound <= round(nominal_flow, 4) <= upper_bound
+
+
+def run_governance_assimilation(ssot_path: str, active_flow: float, active_efficiency: float, mass_mix: list):
+    """Parses incoming SSOT parameters from the federated plane and enforces runtime compliance gates."""
+    print("Executing Predatory Assimilation: Processing G10-TUPLE-HE-REF...")
```
[the remark this answers is not in the document]
Comment (GBOGEB) [reply physics/helium_refrigeration_core.py]: @copilot fix and commit

File: physics/helium_refrigeration_core.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+    try:
+        with open(ssot_path, "r", encoding="utf-8") as f:
+            ssot_data = yaml.safe_load(f)
+    except Exception as e:
+        print(f"Execution Halt: Failed to bind to governance contract mapping layer. Details: {e}")
+        sys.exit(1)
```
[the remark this answers is not in the document]
Comment (GBOGEB) [reply physics/helium_refrigeration_core.py]: @copilot fix and commit

File: physics/helium_refrigeration_core.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+    he_tuple = next((c for c in ssot_data["ssot"]["components"] if c["id"] == "G10-TUPLE-HE-REF"), None)
+    if not he_tuple:
+        print("CRITICAL FAULT: Component identifier G10-TUPLE-HE-REF missing from runtime substrate.")
+        sys.exit(1)
+
+    target_eff = he_tuple["modes"]["2K-SB"]["target_efficiency"]
+    target_flow = he_tuple["modes"]["2K-OP"]["nominal_flow_g_s"]
+
```
[the remark this answers is not in the document]
Comment (GBOGEB) [reply physics/helium_refrigeration_core.py]: @copilot fix and commit

File: physics/helium_refrigeration_core.py [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+if __name__ == "__main__":
+    mock_mass_fractions = [0.9995, 0.0005]
+    simulated_flow = 11.4850
+    simulated_efficiency = 0.3620
+
+    run_governance_assimilation(
+        ssot_path="SSOT/g10_runtime_governance_ssot.yaml",
+        active_flow=simulated_flow,
```
[the remark this answers is not in the document]
Comment (GBOGEB) [reply physics/helium_refrigeration_core.py]: @copilot fix and commit

File: physics/helium_refrigeration_core.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (GBOGEB) [reply physics/helium_refrigeration_core.py]: @copilot fix and commit
